errors.txt

```diff
@@ -0,0 +1,202 @@
+15) CodeIgniter\Files\FileCollectionTest::testResolveDirectorySymlink
+ErrorException: symlink(): Permission denied
+
+E:\www\CodeIgniterProjects\dev\tests\system\Files\FileCollectionTest.php:69
+
+16) CodeIgniter\Files\FileCollectionTest::testResolveFileSymlink
+ErrorException: symlink(): Permission denied
+
+E:\www\CodeIgniterProjects\dev\tests\system\Files\FileCollectionTest.php:89
+
+17) CodeIgniter\HTTP\Files\FileMovingTest::testStore
+ErrorException: mkdir(): No such file or directory
+
+E:\www\CodeIgniterProjects\dev\system\HTTP\Files\UploadedFile.php:181
+E:\www\CodeIgniterProjects\dev\system\HTTP\Files\UploadedFile.php:135
+E:\www\CodeIgniterProjects\dev\system\HTTP\Files\UploadedFile.php:355
+E:\www\CodeIgniterProjects\dev\tests\system\HTTP\Files\FileMovingTest.php:213
+
+18) CodeIgniter\Helpers\FilesystemHelperTest::testGetFilenamesWithSymlinks
+ErrorException: symlink(): Permission denied
+
+E:\www\CodeIgniterProjects\dev\tests\system\Helpers\FilesystemHelperTest.php:408
+
+19) CodeIgniter\Helpers\FilesystemHelperTest::testGetFileNotThereInfo
+TypeError: basename(): Argument #1 ($path) must be of type string, false given
+
+E:\www\CodeIgniterProjects\dev\system\Helpers\filesystem_helper.php:322
+E:\www\CodeIgniterProjects\dev\tests\system\Helpers\FilesystemHelperTest.php:525
+
+20) CodeIgniter\Publisher\PublisherInputTest::testAddUri
+CodeIgniter\HTTP\Exceptions\HTTPException: 60 : SSL certificate problem: unable to get local issuer certificate
```

Comment:
It seems that this issue is related to an incorrect configuration on your system. I do not have this problem. Please try to resolve it by referring to the link below:

https://stackoverflow.com/questions/35638497/curl-error-60-ssl-certificate-prblm-unable-to-get-local-issuer-certificate

Reply:
Ok. I used to disable validation in other scripts. I didn't get an error. If the PR is approved, it will be described in the documentation